src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/TableParsers/DataLockEventsTableParser.cs

```diff
@@ -46,7 +46,7 @@ private static DataLockEventsTableColumnStructure ParseDataLockEventsTableStruct
                     case "ilr training price":
                         structure.IlrTrainingPriceIndex = c;
                         break;
-                    case "ilr ed point assessment price":
```

Comment:
Could the for loop in this and the CommitmentsTableParser class be simplified to reduce complexity?

Reply:
line 74 - should be:

`  var effectiveFrom = row.ReadRowColumnValue<DateTime>(structure.IlrEffectiveFromIndex, "ILR Effective from", startDate);`

Reply:
line 84 - comma at end of line not required